packages/vaadin-virtual-list/src/virtualizer.js

```diff
@@ -1,4 +1,4 @@
-import { IronListAdapter } from './vaadin-virtualizer-iron-list-adapter';
+import { IronListAdapter } from './virtualizer-iron-list-adapter';
```

Comment:
Let's add `.js` extension for consistency.

Reply:
Done